Github page updated and added banner - April 2026 (#29)
espressif/.github: profile/README.md

![Espressif GitHub Banner](github-banner.png)

> # Welcome to Espressif's site on GitHub

Reviewer: The possessive case interferes with the company name readability. I suggest that you remove it:

```suggestion
# Welcome to Espressif on GitHub
```

Espressif supports a large variety of open-source projects, including SDKs, components, libraries, solutions, and tools, which aim to help developers bring their projects to life.

All of Espressif’s official software, relating to the various series of ESP SoCs including ESP32 and ESP8266, are available on this GitHub site.
To check out all the series of SoCs from Espressif, please visit our [ESP Product Selector](https://products.espressif.com).

Below you can find a selection of Espressif’s open-source projects. Our full repository list can be found [here](https://github.com/orgs/espressif/repositories).

## Get Started

To get started with our solutions, visit our [Developer Portal](https://developer.espressif.com/). The Developer Portal includes articles about solutions, announcements, latest releases, workshops, and events.

> For the [ESP-IDF](https://developer.espressif.com/tags/esp-idf/), you can follow this [workshop](https://developer.espressif.com/workshops/esp-idf-basic/).

Reviewer: ```suggestion
For the [ESP-IDF](https://developer.espressif.com/esp-idf/), you can follow this [workshop](https://developer.espressif.com/workshops/esp-idf-basic/).
```

> If you are looking for the official ESP-IDF documentation, click [here](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/index.html) and select the ESP32 series you are interested in from the dropdown menu in the top left corner.

Reviewer: You can also check the official [ESP-IDF documentation](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/index.html). In the top left corner, find the drop-down menu and select the appropriate ESP32 series.

If you have questions, ask our [AI chatbot](https://chat.espressif.com/).

## Frameworks

> Espressif's core development frameworks provide the foundation for building applications on ESP32 and ESP8266, supporting both native and Arduino-based development.

Reviewer: > ESP32 and ESP8266

What about other series?

I suggest that you avoid the possessive case:

```suggestion
Espressif core development frameworks provide the foundation for building applications on ESP32 and ESP8266, supporting both native and Arduino-based development.
```

| Category | Project |
|---|---|
| AT Commands Firmware | [ESP-AT](https://github.com/espressif/esp-at) |
| Communication Processor Firmware | [ESP-HOSTED](https://github.com/espressif/esp-hosted) |
| Cloud | [ESP-RainMaker](https://github.com/espressif/esp-rainmaker) |
| Smart Devices | [ESP-Matter](https://github.com/espressif/esp-matter) |
| Connectionless Wi-Fi Communication Protocol | [ESP-NOW](https://github.com/espressif/esp-now) |
| HMI | [ESP-Brookesia](https://github.com/espressif/esp-brookesia) |
| Audio | [ESP-ADF](https://github.com/espressif/esp-adf) |

## AI

Espressif provides AI-focused libraries and frameworks to enable machine learning, signal processing, and intelligent agent capabilities on ESP devices.

| Category | Project |
|---|---|
| AI Agent Framework | [ESP-CLAW](https://github.com/espressif/esp-claw) |
| High-Performance Deep Learning | [ESP-DL](https://github.com/espressif/esp-dl) |
| Digital Signal Processing | [ESP-DSP](https://github.com/espressif/esp-dsp) |

### MCP Servers

If you are looking for the MCP servers, please visit: https://mcp.espressif.com/

## Libraries and Components

> Espressif and the community maintain a rich ecosystem of reusable libraries and components for ESP devices. Explore thousands of components — from drivers and protocols to UI libraries and cloud integrations — right on the [ESP Registry](https://components.espressif.com/).

Reviewer: ```suggestion
Espressif and the community maintain a rich ecosystem of reusable libraries and components for ESP devices. Explore thousands of components — from drivers and protocols to UI libraries and cloud integrations — right on the [ESP Component Registry](https://components.espressif.com/).
```

## More

> Our full repository list can be found [here](https://github.com/orgs/espressif/repositories).

Reviewer: It is generally recommended to avoid "click here", see "this page", because the highlighted link text does not give any clues where it will take you. You need to look for this information in the surrounding text, which is suboptimal.

```suggestion
See also our [full repository list](https://github.com/orgs/espressif/repositories).
```

> To know more about our frameworks, solutions and libraries, see this brief explanation for some of our projects: [Espressif Projects](esp-projects.md).

Reviewer: ```suggestion
To know more about our frameworks, solutions and libraries, see the brief explanation for some of our projects on the [Espressif Projects](esp-projects.md) page.
```
